### ISSUE TITLE
Review and update content about SameSite cookies

### ISSUE THREAD

**Issue by rbsec** [OWASP/wstg#1005]
There are few areas of the guide that don't really take into account the current state of `SameSite` cookies:

* The Testing Cookie Attributes guide says that "most" browsers" default to `Lax`, but this isn't true for Firefox or Safari.
* The Cross-Site Request Forgery guide doesn't mention it at all, and the PoCs given won't work in *some* browsers because of it.
* The Cross-Site Script Inclusion guide doesn't mention it at all, and the PoC given won't work in *some* (all? #954) browsers because of it.

There might also be some other areas that I've missed?

**Comment by mademarc** [comment]
Hey @rbsec Can in contribute and help out on this issue that needs revise?  

Please let know when possible

**Comment by kingthorin** [comment]
Absolutely. Just let us know if you have questions or if there's enough detail there to get you started.

**Comment by mademarc** [comment]
Thanks @kingthorin and i just message you some questions via slack, did you received them?

**Comment by Corencas** [comment]
Can I be added to this issue? I recently contributed to OWASP/wstg (#1145) and would like to help here.

Let me know if there's any specific direction you'd like me to take.

**Comment by kingthorin** [comment]
Just keep in mind the issue is 3y old so even the original comment(s) are out of date. You'll need to research actual current state/behaviour of browsers.

**Comment by workmcg** [comment]
Hi all — hope this is helpful for whoever's picking this up.

Worth flagging: the "most browsers default to Lax" framing needs a correction, not just a caveat. Chromium-based browsers (Chrome, Edge, Opera) have treated unset SameSite cookies as Lax since Chrome 80 (Feb 2020). Firefox and Safari don't do this — they leave SameSite unenforced by default and instead rely on Enhanced Tracking Protection / Total Cookie Protection (Firefox) and Intelligent Tracking Prevention (Safari) to restrict third-party cookies. Those are privacy/anti-tracking mechanisms, not CSRF defenses, and they behave differently from an explicit SameSite=Lax/Strict attribute.

Practical implication for testers: a CSRF PoC that "fails" in Firefox/Safari because of ITP/TCP can give a false sense of security — the app may still have no SameSite protection at all. Testers should verify the actual Set-Cookie header rather than inferring protection from browser behavior.

Happy to help draft wording for the Testing Cookie Attributes and CSRF sections if useful — thanks for the work on this guide!

**Comment by kingthorin** [comment]
@workmcg always happy to have new contributors.

**Comment by kingthorin** [comment]
@copilot do you see anything outstanding related to this issue and the associated content?
